conf/messages

```diff
@@ -943,7 +943,10 @@ returns.property-details.multipleDisposals.caption=Details of one property dispo
 property-details.multiple-disposals.guidance.title=You’ll need to enter details for one of the property disposals
 property-details.multiple-disposals.guidance.p1=You should enter the details of the property that gave you the largest gain or the smallest loss.
 property-details.multiple-disposals.guidance.agent.p1=You should enter the details of the property that gave your client the largest gain or the smallest loss.
+property-details.multiple-disposals.guidance.p1=You should enter the details of the property that gave the trust the largest gain or the smallest loss.
```

Comment:
this looks like a duplicate key you've added with what looks like the wrong content for it - it should be safe to delete this one